### ISSUE TITLE
ver. 0.14.10_2 - totally broken?

### ISSUE THREAD

**Issue by luckman212** [spamwax/alfred-pinboard-rs#27]
At first I tried updating from my older version, and that failed, so I completely deleted the workflow and downloaded a fresh copy and started over.

Still, when I type `pconf` or `ps {query}` I just get this error "Config file may be corrupted":
![image](https://user-images.githubusercontent.com/1992842/52792233-d9aff500-3038-11e9-8273-ce496488fabd.png)

Alfred v3.8 [959] on macOS 10.14.3

**Comment by luckman212** [comment]
Also, notifications are appearing strangely, looks like they are just outputting raw JSON:
![image](https://user-images.githubusercontent.com/1992842/52792733-1defc500-303a-11e9-942d-404ac0e76c77.png)

**Comment by spamwax** [comment]
you can copy content of your `settings.json` from `~/Library/Application Support/Alfred 3/Workflow Data/cc.hamid.alfred-pinboard-rs` folder here. Make sure to delete your authorization code before doing so.

**Comment by luckman212** [comment]
Sure, here it is:

```text
{"pins_to_show":10,"tags_to_show":10,"tag_only_search":false,"fuzzy_search":false,"private_new_pin":true,"toread_new_pin":false,"suggest_tags":false,"auto_update_cache":true,"auth_token":"luckman212:xxxxxxxxxxxxxxxxxxxx","update_time":"2019-01-15T22:46:37.318701Z","workflow_data_dir":"/Users/luke/Library/Application Support/Alfred 3/Workflow Data/cc.hamid.alfred-pinboard-rs","workflow_cache_dir":"/Users/luke/Library/Caches/com.runningwithcrayons.Alfred-3/Workflow Data/cc.hamid.alfred-pinboard-rs"}
```

**Comment by spamwax** [comment]
This is getting interesting!
Can you open the debug window of this workflow in Alfred, then run `pu` and post the output please?
If that works and you get the cache updated, then try to post a new bookmark to Pinboard and see if you get a notification for successful cache update.

![screen shot 2019-02-15 at 5 37 05 pm](https://user-images.githubusercontent.com/1251233/52888097-5f6e9600-3148-11e9-9653-524554ad48c0.png)

If not, I will have to ask you to do more debugging.
Thanks for helping to figure this out.

**Comment by suaff** [comment]
Same problem here, 0.14.9 is the last working without that error

**Comment by spamwax** [comment]
@suaff please try to uninstall the workflow remove `settings.json` file from `~/Library/Application Support/Alfred 3/Workflow Data/cc.hamid.alfred-pinboard-rs` and re-setup the workflow.

If that doesn't help please follow what I suggested luckman212 in my previous post.
In order for me to fix it, I'm gonna need more info.

**Comment by luckman212** [comment]
Sorry for the late reply- but I wanted to post that ver 0.14.11 fixed these issues for me. I deleted the settings & cache from:
```
~/Library/Caches/com.runningwithcrayons.Alfred-3/Workflow Data/cc.hamid.alfred-pinboard-rs
~/Library/Application Support/Alfred 3/Workflow Data/cc.hamid.alfred-pinboard-rs
```
Then downloaded the latest workflow and reinstalled, all working fine now. Thank you so much. 👍

**Comment by bdubn** [comment]
FYI. In my case, I was never getting "config file may be corrupted". However, I was getting same json output in notification and 1970 for latest cache update. None of the steps below solved my issue. I had recently destroyed all of my bookmarks and therefor tags from pinboard to clean out and start over. That is what caused the issue for me. Even though I added a a single bookmark back to pinboard so that rustypin would have some data to cache, I did not set a tag and had zero tags from the previous cleanout. Adding one tag resolved my issue. I was then able to "pu" successfully, verify latest update, and json in notifications went away. So in theory, the issue was due to the status of pinboard itself and not any error within rustypin although maybe logic needs to be in the rustypin code to deal with an empty pinboard that has no tags. Hope this helps improve code logic. Thanks!
